js/app.js

```diff
@@ -7,102 +7,48 @@ function SetStore(name, minHourlyCustomers, maxHourlyCustomers, avgCookiesPerCus
   this.minHourlyCustomers = minHourlyCustomers;
   this.maxHourlyCustomers = maxHourlyCustomers;
   this.avgCookiesPerCustomer = avgCookiesPerCustomer;
-  // this.cookiesPurchased = function cookiesPurchased() {
-  //   return Math.floor(this.avgCookiesPerCustomer * this.customersPerHour());
-  // };
-  // this.customersPerHour = function customersPerHour() {
-  //   return (Math.random() * (this.maxHourlyCustomers - this.minHourlyCustomers + 1)) + this.minHourlyCustomers;
-  // };
-  // this.locationCookies = function locationCookies() {
-  //   for (var i = 0; i < 15; i++) {
-  //     Location.push(this.cookiesPurchased())[i];
-  //     totale += Location[i];
-  //   }
-  // };
 }
 
 // to get the number of cookies purchased per hour
-
 SetStore.prototype.cookiesPurchased = function cookiesPurchased() {
   return Math.floor(this.avgCookiesPerCustomer * this.customersPerHour());
 };
 
 //to get the number of customers per hour
-
 SetStore.prototype.customersPerHour = function customersPerHour() {
   return (Math.random() * (this.maxHourlyCustomers - this.minHourlyCustomers + 1)) + this.minHourlyCustomers;
 };
 
 
-var total = 0;
-Location = [];
-
 // adding the TOTAL number of cookies to an array
+Location = [];
 SetStore.prototype.locationCookies = function locationCookies() {
   for (var i = 0; i < 15; i++) {
```

Comment:
You should avoid hardcoding how many times your for loop iterates. If your cookiestore hours change or amount of locations change your code won't be scalable.